private/pkg/protosource/field.go

```diff
@@ -207,3 +210,7 @@ func (f *field) PackedLocation() Location {
 func (f *field) ExtendeeLocation() Location {
 	return f.getLocation(f.extendeePath)
 }
+
+func (f *field) PresentFields() map[protoreflect.FieldDescriptor]protoreflect.Value {
+	return f.optionExtensionDescriptor.PresentFields()
+}
```

Comment:
You should be able to remove this. `optionExtensionDescriptor` is embedded, so this method is present automatically by virtue of adding it to `optionExtensionDescriptor`.